release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4555,4 +4555,32 @@ $ oc adm release info 4.13.58 --pullspecs
 
 [id="ocp-4-13-58-updating_{context}"]
 ==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.59
+[id="ocp-4-13-59_{context}"]
+=== RHSA-2025:10747 - {product-title} 4.13.59 bug fix and security updates
+
+Issued: 17 July 2025
+
+{product-title} release 4.13.59, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:10747[RHSA-2025:10747] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:10748[RHBA-2025:10748] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.59 --pullspecs
+----
+
+[id="ocp-4-13-59-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, an {product-title} upgrade bug from product version 4.12 to 4.13, caused network policies to incorrectly block traffic that was supposed to be allowed on the host network. Because of this bug, the pods on different nodes could not communicate with each other. With this release, during upgrades from 4.12 to 4.14, network policies work correctly, and pods communicate across different nodes. (link:https://issues.redhat.com/browse/OCPBUGS-57051[OCPBUGS-57051])
```

Comment:
```suggestion
* Before this update, an {product-title} upgrade bug from product version 4.12 to 4.13 caused network policies to incorrectly block traffic that was supposed to be allowed on the host network. Because of this bug, the pods on different nodes could not communicate with each other. With this release, during upgrades from 4.12 to 4.14, network policies work correctly, and pods communicate across different nodes. (link:https://issues.redhat.com/browse/OCPBUGS-57051[OCPBUGS-57051])
```
comma splice but not a merge blocker; optional fix